src/Tizen.NUI.Scene3D/src/public/ModelMotion/MotionIndex/MotionTransformIndex.cs

```diff
@@ -122,7 +122,7 @@ public MotionTransformIndex() : this(Interop.MotionIndex.MotionTransformIndexNew
         /// <summary>
         /// Create an initialized blend shape index with input node id, and transform type.
```

Comment:
blend shape again, there are 4 of those outside of BlendShapeIndex.cs file

Reply:
Copy & paste mistakes! Fix done.